### DISCUSSION TITLE
Интеграция с библиотекой

### DISCUSSION THREAD

**Discussion by TemofeyF** [quarckster/kodi.kino.pub#195] [Ideas]
Добрый день! Совсем недавно открыл для себя kino.pub, в том числе через ваш плагин для kodi. Большое спасибо вам за это! Я давно пользуюсь kodi как основным медиакомбайном и мне очень не хватает в вашем плагине интеграции в библиотекой kodi на подобии Elementum с привязкой к аккаунту trakt.tv (это очень удобно). Это ускоряет интерфейс при ежедневном просмотре сериалов и фильмов. А также выглядит лучше за счёт более подробных описаний. Может быть это можно сделать через Tmdb helper, добавив kino.pub в players? Очень надеюсь, что вы поймёте о чем я))) я сам в этом не разбираюсь, но очень хочется чтобы это появилось в плагине. Потому что сейчас приходится пользоваться kodi 18.9, так как плагина add to lib нет на 19 версии. А других способов добавления ссылок strm из плагина в библиотеку не знаю. Искренне надеюсь, что мои идеи реализуемы. Спасибо вам большое за плагин!

**Comment by Kampfader** [comment]
+1 от меня.

**Comment by quarckster** [comment]
Я не заинтересован в данной фиче. Если есть желающие реализовать, то шлите пулл реквесты.

**Comment by TemofeyF** [comment]
Добрый день! Помогите, пожалуйста (если конечно Вас это не затруднит) сделать Player File для плагина TMDB helper. Я к сожалению не программист, но с вашей помощью может быть что-нибудь получится)))
Есть инструкция по созданию этого файла: https://github.com/jurialmunkey/plugin.video.themoviedb.helper/wiki/Player-Function
Я попробовал на примере плагина Netflix:
{
    "name"              : "kino.pub",
    "plugin"            : "video.kino.pub",
    "play_movie"        : [
                            "plugin://video.kino.pub/directory/search/search/{title}/",
                            {"title": "{title}", "year": "{year}"}
                          ],
    "play_episode"      : [
                            "plugin://video.kino.pub/directory/search/search/{showname}/",
                            {"title": "{showname}"},
                            {"position": "{season}"},
                            {"season": "{season}", "episode": "{episode}"}
                          ],
    "search_movie"      : "plugin://video.kino.pub/directory/search/search/{title}/",
    "search_episode"    : "plugin://video.kino.pub/directory/search/search/{showname}/"
}

Не получается. Понимаю, что нужно правильно прописать путь, но не знаю где его искать.
